packages/create-instance/create-component-stubs.js

```diff
@@ -134,24 +112,19 @@ export function createStubFromComponent(
     render(h, context) {
       return h(
         tagName,
-        {
-          ref: componentOptions.functional ? context.data.ref : undefined,
```

Comment:
are all these included in `context.data.attrs`? Curious why so much code here can be removed.

Reply:
This is my main concern - since we are in 1.0 now, this is a *breaking* change for a lot of users. But it does make sense we have feature parity with Vue.

Maybe @afontcu has some opinion? My guess is this will only impact snapshots users - so if we make it clear they need to update their snapshots, maybe it's okay.

Reply:
I agree that the better way is to request for comments of other developers and not to push breaking changes now.

Therefore, I kept the standard behavior VTU and only add support array class binding and events binding into stubbed functional components.

Reply:
Right I think I misunderstood - just to clarify, this change only impacts stubbed functional components (and no change to non stubbed components, and regular stubbed components?)

If that is the case this probably isn't as big a change as I thought, and we could just make this a minor version release. Can you please clarify?

Reply:
Yes, this change only impacts stubbed functional components. 

I was planning this as a MINOR update, maybe even as a PATCH.

Reply:
I think this is fine. I will make this a minor release.